### PR TITLE
fix: remove old license file

### PR DESCRIPTION
**Pull request by arys-splunk**
This is a very late followup for https://github.com/splunk/addonfactory-repository-template/pull/84

### CODE REVIEW

File: tools/sync.sh [hunk 1]
```diff
@@ -324,6 +324,10 @@ else
         git rm semtag
     fi
 
+    if [[ -f "LICENSES/LicenseRef-Splunk-1-2020.txt" ]]; then
```
Comment (uoboda-splunk) [inline tools/sync.sh:327]: I am not aware how these licences work exactly but I think in some TAs we still refer to that license. Wouldn't it be a problem?

Comment (arys-splunk) [reply tools/sync.sh:327]: I mean, ideally they should not be using this license, because it is an old one.

Comment (arys-splunk) [reply tools/sync.sh:327]: And I hope they'll get reuse failure because they refer to the old license :)

Comment (uoboda-splunk) [reply tools/sync.sh:327]: ok, so it is forcing to use new one. I think we can go with it

**Review by uoboda-splunk — APPROVED** [review]

**Pull request merged**